inscriptions/ISic0999.xml

```diff
@@ -173,30 +178,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140477">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— —</supplied>
-                    ζὸν σχὸν ὕδωρ
-                    <lb n="2"/>
-                    <supplied reason="lost">— —</supplied>
-                    εἰς μνημόσυνον
-                    <lb n="3"/>
-                    <supplied reason="lost">— —</supplied>
-                    αἰώνιον ὑπὲρ εὐχῆ
-                    <supplied reason="lost">ς</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— —</supplied>
-                    τῶν δεσπότων αὐτοῦ
-                    <lb n="5"/>
-                    <supplied reason="lost">— —</supplied>
-                    Ὑγεῖνος <supplied reason="omitted">ὁ</supplied> καὶ <unclear>Γ</unclear>εν<unclear>
-                    ν</unclear>ά<unclear>δ</unclear>ις.
-                    <lb n="6"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>ζ<choice><corr>ῶ</corr><sic>ο</sic></choice>ν ἔχ<choice><corr>ω</corr><sic>ο</sic></choice>ν ὕδωρ                    
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/> εἰς μνημόσυνον
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/> αἰώνιον ὑπὲρ εὐχῆ<supplied reason="lost">ς</supplied>                   
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/> τῶν δεσποτῶν αὐτοῦ
+                    <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/> Ὑγεῖνος κα<supplied reason="omitted">ὶ</supplied> <orig>ΠΣΝΥΑΔΙΣ</orig>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text of IG with slight changes based on Kaibel's diplomatic transcription</note>
```

Comment:
For the text on the other side (ISic1004), simply note it in the commentary for now